### DISCUSSION TITLE
Remove entities and colliders label in ursina window.

### DISCUSSION THREAD

**Discussion by Sonarjit** [pokepetter/ursina#570] [Q&A] [answered]
How to remove "entities" and "colliders" label at right top corner in ursina window? I have already disable fps_counter, entity_counter and collider_counter. Counter numbers are not shown, but the labels are still showing. How to disable the labels also?
```
app = Ursina(title="Vajra", borderless=False, fullscreen=False)

# ursina diable fps_counter
window.fps_counter.enabled = False
# ursina disable entity_counter
window.entity_counter.enabled = False
# ursina disable collider_counter
window.collider_counter.enabled = False
# ursina disable cog_button
window.cog_button.enabled = False
```
**THANKS**

**Comment by a2cy** [comment, marked as answer]
you could just disable development mode
this hides all of this

**Reply by Sonarjit** [reply]
Thanks for the response, I have achieved the task with:
`window.editor_ui.disable()`
